js/client.js

```diff
@@ -1158,6 +1162,18 @@ function toId() {
 				break;
 			}
 		},
+		saveIgnore: function () {
+			Storage.prefs('ignorelist', Object.keys(this.ignore));
+		},
+		loadIgnore: function () {
+			var data = Storage.prefs('ignorelist');
+			if (!data) return {};
+			var buf = {};
+			for (var i = 0; i < data.length; i++) {
+				buf[data[i]] = {notified: false};
```

Comment:
I think we'd be better off with one overall `ignoreNotified` flag, rather than a per-user flag? I think this would make it easier on people with a long ignore list.